### DISCUSSION TITLE
Storing collections in different directory

### DISCUSSION THREAD

**Discussion by Anwenya** [qdrant/qdrant#5410] [Q&A]
I just use a standalone mode on my windows，is there any way to specify that the collections be stored in a different directory?  this means that each collection has its own storage_path parameter。 thanks！

**Comment by timvisee** [comment]
No, you cannot change this per collection. You can only specify a directory in which each collection will be stored.

Is there a specific reason why you'd like to be able to do this?

**Reply by Anwenya** [reply]
In our product, users can create any business flow, and the data of each business flow is stored together, such as a folder, which is convenient for backing up and archiving individual business flows, and can also directly move the entire directory to other machines for continued use. Therefore, the vector data of each business flow must also be stored in its own directory.

We are now trying to solve this problem from a business perspective, thank you again

**Reply by timvisee** [reply]
> and can also directly move the entire directory to other machines for continued use

That will not work in Qdrant. It is not possible to simply move a directory to another machine to host it there. Nor will it be possible to use collection directory backups from different points in time.

All collections in a Qdrant instance should remain together, and should be backed up in one shot.

This is required to achieve consensus across nodes.

**Reply by Anwenya** [reply]
Got it. Since I'm not currently using the distributed version, I'm wondering if there is a way to store them separately. I konw the reason now，tanks
